### PR TITLE
Community Manager Announcement Post

### PR DESCRIPTION
**Pull request by austinlparker**
This is a blog post to announce the new community manager role.

### CODE REVIEW

File: content/en/blog/2022/announcing-community-manager.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Community managers are appointed by the Governance Committee and act as public stewards of the contributor and end-user community. They’re responsible for organizing events, coordinating cross-SIG and WG efforts to improve contributor experience, managing the OpenTelemetry presence on social media, and working to nurture and grow the OpenTelemetry community overall. Effectively, this is a new project maintainer whose project is the OpenTelemetry community itself.
+
+With that said, I’m happy to inform you that I have been designated the first Community Manager for the OpenTelemetry project! As a former OpenTracing maintainer, I’ve been a part of OpenTelemetry since its inception, mostly working on projects such as the website, hosting OpenTelemetry Tuesdays, and helping organize events like OpenTelemetry Community Day. With this new role, I plan to more formally establish OpenTelemetry Community Days around the world as part of KubeCon/CloudNativeCon -- and additionally, help promote new end-user and contributor events like OTel Unplugged (more on that in a week or so.)
```
Comment (cartermp) [inline content/en/blog/2022/announcing-community-manager.md:14]: Might be cool to have a picture of your face or something here to add a 'lil more human element to it

**Review by cartermp — APPROVED** [review]

File: content/en/blog/2022/announcing-community-manager.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing the OpenTelemetry Community Manager
+linkTitle: Announcing OpenTelemetry Community Manager
+date: 2022-09-24
+author: Austin Parker
```
Comment (svrnm) [inline content/en/blog/2022/announcing-community-manager.md:5]: Point to your github profile (or twitter handle?) here

File: content/en/blog/2022/announcing-community-manager.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+OpenTelemetry has demonstrated massive growth since its inception in 2019. What started as a handful of OpenTracing and OpenCensus maintainers and collaborators meeting at the Google campus and over Zoom has now grown into the second-most popular project in the CNCF behind Kubernetes itself. Over 5400 contributors and 700 companies have contributed code, issues, documentation, and invaluable feedback.
```
Comment (svrnm) [inline content/en/blog/2022/announcing-community-manager.md:8]: > What started as a handful of OpenTracing and OpenCensus maintainers and collaborators meeting at the Google campus and over Zoom has now grown into the second-most popular project in the CNCF behind Kubernetes itself.

Put a `,` behind Zoom? Not sure if this works in English, but it helps the reading flow to understand that the injection ends here (I got confused why OTel has grown over zoom...)

Comment (svrnm) [inline content/en/blog/2022/announcing-community-manager.md:8]: > Over 5400 contributors and 700 companies have contributed code, issues, documentation, and invaluable feedback. 

Can you link to the source of that data? I assume CNCF dev stats?

File: content/en/blog/2022/announcing-community-manager.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+What’s next? I hope to hear from you about what you need! I’ve started a [thread](https://github.com/open-telemetry/community/discussions/1203) on the Community Discussions in GitHub, and I’d love for you to post your questions, comments, and suggestions for community initiatives and programs. This isn’t just limited to end-users of OpenTelemetry, either -- how can I help in improving the existing and new contributor experience? Please let me know! 
+
+If you’d like to reach out in other ways, you can find me on [Twitter](https://twitter.com/@austinlparker) or on the Cloud Native Community Slack.
```
Comment (svrnm) [inline content/en/blog/2022/announcing-community-manager.md:18]: > Cloud Native Community Slack.

Add a link to https://slack.cncf.io/, maybe also add a specific channel (#opentelemetry?) where people can find you.

**Review by svrnm — CHANGES_REQUESTED** [review]
I added some suggestions :-)

File: content/en/blog/2022/announcing-community-manager.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing the OpenTelemetry Community Manager
+linkTitle: Announcing OpenTelemetry Community Manager
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/blog/2022/announcing-community-manager.md:3]: I also submitted a change to fix the build.

File: content/en/blog/2022/announcing-community-manager.md (hunk 4 above)
Comment (austinlparker) [reply content/en/blog/2022/announcing-community-manager.md]: Done

File: content/en/blog/2022/announcing-community-manager.md (hunk 3 above)
Comment (austinlparker) [reply content/en/blog/2022/announcing-community-manager.md]: Done

Comment (austinlparker) [reply content/en/blog/2022/announcing-community-manager.md]: Done

File: content/en/blog/2022/announcing-community-manager.md (hunk 2 above)
Comment (austinlparker) [reply content/en/blog/2022/announcing-community-manager.md]: Done

File: content/en/blog/2022/announcing-community-manager.md (hunk 1 above)
Comment (austinlparker) [reply content/en/blog/2022/announcing-community-manager.md:14]: eh it'd be weird if it was just a big picture of me in there.

**Comment by austinlparker** [comment]
comments addressed ptal

**Review by chalin — APPROVED** [review]
🎉

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
Merged :-)

**Pull request merged**